packages/sdk/src/evm/core/classes/erc-20-history.ts

```diff
@@ -55,14 +55,24 @@ export class TokenERC20History {
         balances[to] = balances[to].add(amount);
       }
     });
+    const _promises = Object.entries(balances).map(async ([addr, value]) => {
```

Comment:
do we have a unit test for this one?

Reply:
I think we do: https://github.com/thirdweb-dev/js/blob/main/packages/sdk/test/evm/token.test.ts#L65